server/modules/blendsubs/word-obfuscation/word-obfuscation.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function(word) {
```

Comment:
I think we don't need to repeat the folder+file here (e.g /word-obfuscation/word-obfuscation.js), maybe put them inside a feature|replacement folder.